src/main/java/org/geysermc/discordbot/commands/moderation/ForumPostCommand.java

```diff
@@ -147,6 +147,7 @@ public void onAutoComplete(CommandAutoCompleteInteractionEvent event) {
             List<ForumTag> tags;
             try {
                 ForumChannel forumChannel = ServerSettings.getForumChannel(Objects.requireNonNull(event.getGuild()));
+                Checks.check(forumChannel != null, "Forum channel not found");
```

Comment:
This will fail silently (as in, user will just get a "Command failed" error) unless you catch `IllegalArgumentException` too

Reply:
The command executor checks for the forum channel being null (in theory), but for safety i added this exception